jwt/audience.go

```diff
@@ -22,43 +21,25 @@ func NewAudience(u string) *Audience {
 // Validate performs error checking on the Audience's URL.
 func (aud *Audience) Validate() error {
 	rawAudience := string(*aud)
-	p := strings.SplitN(rawAudience, "/", 5)
-	if len(p) != 5 {
-		return errors.New("invalid audence length")
+	p := strings.SplitN(rawAudience, "/", 4)
+	if len(p) != 4 {
+		return errors.New("invalid audience length")
```

Comment:
`"audience %q must follow the format \"/projects/PROJECT_NUMBER/SERVICE_DETAILS\"", rawAudience`